### PR TITLE
fix: self-closing xhtml

### PR DESCRIPTION
**Pull request by TomWoodward**

### CODE REVIEW

File: src/app/content/components/Page.tsx [hunk 1]
```diff
@@ -13,9 +13,11 @@ export default class PageContent extends Component<PropTypes> {
   public getCleanContent = () => {
     const {content} = this.props;
     return content
+      // remove body and surrounding content
       .replace(/^[\s\S]*<body.*?>|<\/body>[\s\S]*$/g, '')
-      .replace(/<div data-type="document-title">[\s\S]*?<\/div>/g, '')
-      .replace(/<cnx-pi.*>[\s\S]*<\/cnx-pi>/g, '');
+      // fix assorted self closing tags
+      .replace(/<(em|h3|iframe|span|strong|sub|sup|u)([^>]*?)\/>/g, '<$1$2></$1>')
```
Comment (philschatz) [inline src/app/content/components/Page.tsx]: Did none of the tests change? Would it make sense to add a test for these?

Comment (TomWoodward) [reply src/app/content/components/Page.tsx]: ....probably

Comment (TomWoodward) [reply src/app/content/components/Page.tsx]: can we merge the content refs one before i create more merge conflicts in the page.spec?

Comment (TomWoodward) [reply src/app/content/components/Page.tsx]: actually i want some of the utils i made in #48 can we merge that first?

**Review by philschatz — APPROVED** [review]

**Comment by kjdav** [comment]
Passed for me.

**Pull request merged**